platform/posix/transport/utest/openssl_utest.c

```diff
@@ -821,10 +842,16 @@ void test_Openssl_Recv_All_Bytes_Received_Successfully( void )
 {
     int32_t bytesReceived;
 
+    /* No pending data from last read. */
     opensslParams.pSsl = &ssl;
     SSL_read_ExpectAnyArgsAndReturn( BYTES_TO_RECV );
     bytesReceived = Openssl_Recv( &networkContext, opensslBuffer, BYTES_TO_RECV );
     TEST_ASSERT_EQUAL( BYTES_TO_RECV, bytesReceived );
+
+    /* Pending data from last read. */
+    SSL_read_ExpectAnyArgsAndReturn( BYTES_TO_RECV );
```

Comment:
This case seems to be an exact repetition of the case that already exists in this function..?

Reply:
I had meant to change this but got coverage in another function. Changed that here.